.test-infra/validate-runner/build.gradle

```diff
@@ -26,16 +26,12 @@ repositories {
     maven {
         url "https://repo.jenkins-ci.org/releases/"
     }
-    maven {
-        url "https://jcenter.bintray.com/"
-    }
     maven {
         url "https://packages.confluent.io/maven/"
     }
 }
 
 dependencies {
-    implementation group: 'com.cdancy', name: 'jenkins-rest', version: '0.0.22'
```

Comment:
There is no reference to this library in the code so I was able to remove it and remove the jcenter repo because of it.